projects/plugins/jetpack/extensions/blocks/donations/amount/save.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+import formatCurrency, { CURRENCIES } from '@automattic/format-currency';
+
+/**
+ * External dependencies
+ */
+import { minimumTransactionAmountForCurrency } from '../../../shared/currencies';
```

Comment:
I know it's a nitpick, but double check your deps, and sort by `External`, `WordPress`, and `Internal`. 🙂

Reply:
Does there currently exist an automated way of handling this? Maybe we could create an issue to modify the commit hook so that it organizes our imports, otherwise, this will always be susceptible to human error.

Reply:
Done.